app/services/api/v3/finance/statements/index.rb

```diff
@@ -21,6 +21,10 @@ def statements
               ).order(payment_date: :asc)
```

Comment:
chatted with @leandroalemao, we should rename this service

Reply:
Yeah my suggestion is to rename it to `queries.rb` or `actions.rb` because we reusing the same class for different actions on the controller.

Reply:
addressed in https://github.com/DFE-Digital/early-careers-framework/pull/2578/commits/24aa353d56321e570ca59571e3cd9ddaeef79f66